controllers/client/openstackclient_controller.go

```diff
@@ -542,6 +537,8 @@ func (r *OpenStackClientReconciler) SetupWithManager(
 func (r *OpenStackClientReconciler) findObjectsForSrc(ctx context.Context, src client.Object) []reconcile.Request {
 	requests := []reconcile.Request{}
 
+	l := log.FromContext(context.Background()).WithName("Controllers").WithName("PlacementAPI")
```

Comment:
Should `PlacementAPI` be something else here?

Reply:
sure